### ISSUE TITLE
low mean but very significant

### ISSUE THREAD

**Issue by zhanglab2008** [Teichlab/cellphonedb#298]
Hi team,
Great work. I applied CellPhoneDB V2 on my dataset and got very meaningful result. Just one thing has puzzled me for a long time. In the Dot Plot, I saw some interactions having very low log2mean but having very significant p value (p <0.001). How to interpret those interactions? Does it mean that even the expression of the L-R pair in a cell pair is very low but it's still be a significant interaction in that cell pair? 
Thanks in advance!
Zhang

**Comment by eijynagai** [comment]
I have the same question. No matter how small p values I use, still have a large number of genes on my list.

**Comment by D-Bug-dotcom** [comment]
yes, it means the expression of the L/R is low but very specific to this pair of cell types.
